### PR TITLE
Cleaned up the code base with pylint

### PR DESCRIPTION
**Pull request by nishant-jain-94**
1. Cleaned up files which weren't required

2. Cleaned up using pylint Checks

3. Current pylint check score is 9.38

### CODE REVIEW

**Comment by historyofjainism** [comment]
Can we add pycache folder to gitignore?

File: settings.py [hunk 1]
```diff
@@ -65,7 +65,6 @@
                 'django.template.context_processors.request',
                 'django.contrib.auth.context_processors.auth',
                 'django.contrib.messages.context_processors.messages',
-                'views.context_processor'
```
Comment (historyofjainism) [inline settings.py:68]: Why is this removed?

File: settings.py [hunk 2]
```diff
@@ -131,3 +130,14 @@
...
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+
+APP_ENVIRONMENT=os.getenv('APP_ENVIRONMENT')
+
+CLOUDINARY_URL=os.getenv('CLOUDINARY_URL')
+
+CONTENTFUL_URL=os.getenv('CONTENTFUL_URL')
+CONTENTFUL_AUTHORIZATION_TOKEN=os.getenv('CONTENTFUL_AUTHORIZATION_TOKEN')
```
Comment (historyofjainism) [inline settings.py:140]: Could this be a place where we store the keys? https://github.com/historyofjainism/django/settings/secrets/actions

File: settings.py (hunk 1 above)
Comment (nishant-jain-94) [reply settings.py:68]: `views.context_processor` was referring to the following function
```python
def context_processor( request ):
     data = { }
     data['APP_ENVIRONMENT'] = os.environ['APP_ENVIRONMENT']
     data['prod_url'] = PROD_WEBSITE + request.path
     return data
```
1. We first tested if this was being used. Observed that it wasn't. Except in `settings.py`
2. We then checked if its deletion had any impact. Seemed it had no impact on the project.

But please share if we missed something and if this middleware is required. We will be able to refactor it.

**Comment by nishant-jain-94** [comment]
> Can we add pycache folder to gitignore?

`__pycache__` is added but I think an `end_of_line` in the file is missing, hence it's showing the minus sign at the end of it. Will refactor it to add a new line at the end of the `.gitignore` file.
Screenshot 2022-03-27 at 12 11 34 AM

File: settings.py (hunk 2 above)
Comment (nishant-jain-94) [reply settings.py:140]: Yeah. This could be one of the option.
1. Secrets which are created here https://github.com/historyofjainism/django/settings/secrets/actions can only utilised by Github Actions. Github actions is basically a CI/CD Pipeline. So we would end up having a CI/CD Pipeline which deploys this application to the Google Application Engine. Something like this https://github.com/google-github-actions/deploy-appengine/tree/main/.github/workflows. But this doesn't allow the GAE to pull secrets directly from Github Secrets. We would want to potentially update the `app.yaml` as a part of CI Pipeline before the deploy step.
2. One of the other approaches is to do the following https://stackoverflow.com/a/54055525. This basically creates a file but doesn't checks-in to the source code, the secrets file is synced to the GAE by the deployer manually.

File: settings.py (hunk 1 above)
Comment (historyofjainism) [reply settings.py:68]: This is required. Context processor is something that would be called for all views. It would add these parameters to the django template "APP_environment" and prod_url. These are being used in linkedpreview and one other place.

**Comment by historyofjainism** [comment]
It says the PR is still work in progress. Let me know when you are ready for review. I have given my comments so far

**Comment by nishant-jain-94** [comment]
Ready for review.

File: settings.py (hunk 2 above)
Comment (nishant-jain-94) [reply settings.py:140]: Any thoughts on this?

Comment (historyofjainism) [reply settings.py:140]: 1st sounds complicated. 2nd seems better. Where do you think we can store the secrets file though?

File: settings.py (hunk 1 above)
Comment (historyofjainism) [reply settings.py:68]: As mentioned we require this, add the context processor back for merging this commit.

File: settings.py (hunk 2 above)
Comment (nishant-jain-94) [reply settings.py:141]: Sure. To facilitate point 2
1. https://github.com/nishant-jain-94/django/blob/93a0d8597a70f8be6e5c1cf4cd3b2e5cfdd367f9/app.yaml#L13-L14
2. https://github.com/nishant-jain-94/django/blob/09d025832e6781a70b65346505019ac6ad70755f/.gitignore#L69

File: settings.py (hunk 1 above)
Comment (nishant-jain-94) [reply settings.py]: Context Processor is added back. 
1. https://github.com/nishant-jain-94/django/blob/cleanup/views.py#L23-L32
2. https://github.com/nishant-jain-94/django/blob/09d025832e6781a70b65346505019ac6ad70755f/settings.py#L68

**Comment by nishant-jain-94** [comment]
Ready for review.
